app/src/routes.js

```diff
@@ -229,6 +229,9 @@ export default (store) => {
       path: `/feature-not-available/:feature`,
       component: () => import(`./components/common/PageFeatureNotAvailable`),
       props: true,
+      meta: {
+        networkSpecificRoute: true,
+      },
```

Comment:
What are the advantages of having the PageFeatureNotAvailable page being network-specific?

Reply:
when you visit a feature page but the future is not supported (think polkadot governance) it will not change themes or remove the sidebar. routes that are not networkSpecific will use a lunie theme and don't show the sidebar.

Reply:
yes, I got it immediately when you presented it at the meeting